challengeTwo/vowel.js

```diff
@@ -0,0 +1,20 @@
+

```

Comment:
What question were you trying to solve? Put it in a separate pull request and add the question in the description

Reply:
@dennisja this is still challenging